Essay_Analysis_Tool/Notepad-Sharp.csproj

```diff
@@ -130,6 +128,7 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <Folder Include="Interface\" />
```

Comment:
I think this folder no longer required.

Reply:
You are very right, but I was working on this in my spare time at work and had to commit and save quickly before I left lol, sorry about that!

Reply:
Hahaha :smile: